scripts/conda-install.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+# This script installs OpenMC, its Python API, and all necessary dependencies
+# into a conda environment. Because it uses compilers and CMake from the
+# Anaconda repository, it's not necessary to have those installed on your
+# system. However, at a minimum, you do need to have 'make' installed as well as
+# a linker. To get the OpenMC git repository, you'll need git installed as
+# well. On Debian derivatives, you can install all of these with:
+#
+# sudo apt install -y make binutils git
+
+set -e
+
+#===============================================================================
+# INSTALLATION OPTIONS
+
+INSTALL_MINICONDA=yes                             # Install miniconda?
+INSTALL_IN_CONDA=yes                              # Install OpenMC executable/package directly in conda environment?
+INSTALL_EDITABLE=no                               # Install in editable mode (overrides INSTALL_IN_CONDA)
+INSTALL_PREFIX=$HOME/opt/openmc/$(date +%Y-%m-%d) # Where to install (if INSTALL_IN_CONDA and INSTALL_EDITABLE are disabled)
+WITH_MPI=yes                                      # Install with MPI support
+PYTHON_VERSION=3.7                                # What version of Python to use
+ENVNAME=openmc-py$PYTHON_VERSION                  # conda environment name
+
```

Comment:
How about adding an option to install nuclear cross section library, like `INSTALL_XSDATA`? Then scripts `tools/ci/download-xs.sh`  can be executed to download default data library, together with the appropriate environment variables activated? `INSTALL_NJOY` may also be considered.